components/helper/ShareButtons.tsx

```diff
@@ -9,9 +11,133 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    letterSpacing: '.0156rem',
+    textAlign: 'left',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const handleCopyClick = async () => {
```

Comment:
I still had some unexpected behavior using the copy link button. I clicked from the main popup to copy link, and got the link copied popup, then I closed by clicking outside of the modal (NOT USING THE X BUTTON). Then I clicked share the calculator button again and I got the link copied popup instead of the main popup. The only way for me to get the original share screen was to return to link copied, and use the X button. Can you make sure that setCopied returns to false when clicking out of the modal AND when clicking x to close the modal.

Reply:
Oh I just understood what you were talking about. I didn't see the problem you are having because I would always use the X button. I just fixed this.

Reply:
Since I am writing the logic for closing the pop-up at two places should I make a function so I am not being redundant?

Reply:
Yes that is a good idea